File: dayTwo/python/question_two.py
valueOfX = 2
valueOfY = 5
valueOfZ = 3


mean = (valueOfX + valueOfY + valueOfZ) // 3

print(mean)

if(mean == valueOfX){
    print("Mean = x")
}
if(mean == valueOfY){
    print("Mean = y")
}
if(mean == valueOfZ){
    print("Mean = z")
}
    
median = 0

if(valueOfX < valueOfY and valueOfX < valueOfZ):
    if(valueOfY < valueOfZ):
        median = valueOfY
    
    if(valueOfZ < valueOfY):
        median = valueOfZ;
    

if(valueOfY < valueOfX and valueOfY < valueOfZ):
    if(valueOfX < valueOfZ):
        median = valueOfX;
    
    if(valueOfZ < valueOfX):
        median = valueOfZ


if(valueOfZ < valueOfX and valueOfZ < valueOfY):
    if(valueOfY < valueOfX):
        median = valueOfY
    
    if(valueOfX < valueOfY):
        median = valueOfX
    

       
print(median)

if(median == valueOfX){
    print("Median = x");
}
if(median == valueOfY){
    print("Median = y");
}
if(median == valueOfZ){
    print("Median = z");
}

File: dayTwo/Javascript/QuestionTwo.js
let valueOfX = 2;
let valueOfY = 5;
let valueOfZ = 3;


let mean = Math.floor((valueOfX + valueOfY + valueOfZ) / 3);

console.log(mean);

if(mean == valueOfX){
    console.log("Mean = x");
}
if(mean == valueOfY){
    console.log("Mean = y");
}
if(mean == valueOfZ){
    console.log("Mean = z");
}
    
let median = 0;

if(valueOfX < valueOfY && valueOfX < valueOfZ){
    if(valueOfY < valueOfZ){
        median = valueOfY;
    }
    if(valueOfZ < valueOfY){
        median = valueOfZ;
    }
}

if(valueOfY < valueOfX && valueOfY < valueOfZ){
    if(valueOfX < valueOfZ){
        median = valueOfX;
    }
    if(valueOfZ < valueOfX)
        median = valueOfZ;
}

if(valueOfZ < valueOfX && valueOfZ < valueOfY){
    if(valueOfY < valueOfX){
        median = valueOfY;
    }
    if(valueOfX < valueOfY){
        median = valueOfX;
    }
}

console.log(median);
       
if(median == valueOfX){
    console.log("Median = x");
}
if(median == valueOfY){
    console.log("Median = y");
}
if(median == valueOfZ){
    console.log("Median = z");
}
